assets/js/modules/analytics/dashboard/dashboard-widget-popular-pages-table.js

```diff
@@ -80,7 +79,7 @@ class AnalyticsDashboardWidgetPopularPagesTable extends Component {
 		const links = [];
 		const dataMapped = map( data[ 0 ].data.rows, ( row, i ) => {
 			const [ title, url ] = row.dimensions;
-			links[ i ] = siteURL + url;
+			links[ i ] = url;
```

Comment:
Let's prefix these with a slash if they don't start with one. Looks like the `url` values returned don't include that, which looks a bit strange and is also different from how the Analytics frontend renders these paths. Starting in a slash clarifies that this is a path, and also accounts for the case of the home page, where right now this would end up an empty string, when it should rather be just the slash.

Reply:
Done :)